schemas/preferences.sublime-settings.json

```
@@ -855,6 +855,21 @@
           "markdownDescription": "When enabled, clicking on selected text will begin a drag-drop operation.",
           "type": "boolean",
           "default": true
+        },
+        "find_scroll_highlights_limit": {
+          "markdownDescription": "The maximum number of find results to show in the scroll bar. If this number is exceeded no results will be shown. Use `0` to remove this limit.",
```

I think there is a missing comma or `then` here.

```suggestion
          "markdownDescription": "The maximum number of find results to show in the scroll bar. If this number is exceeded then no results will be shown. Use `0` to remove this limit.",
```

(I know this is probably from ST itself but those are not always entirely correct either. :))

---

Taken care of all the points.